### PR TITLE
Test for notifications backend

### LINKED ISSUE #688: Add tests for 'notifications' Backend
## Description

Currently there are no tests for notifications backend, would be nice to get this added

### PR DESCRIPTION
**Pull request by iamhks**
## Summary

Closes #688

## Changes

- Added required tests

### CODE REVIEW

**Comment by iamhks** [comment]
Hi @tuxology I have tried many things from my end, but unable to authorize my test case requests. Would be glad if you could review and suggest changes.
Screenshot 2023-03-26 at 01 22 03

@kamthamc @NdibeRaymond @Ritsyy @khabdrick  Would be great if you all could spare some time as well.

File: zubhub_backend/zubhub/notifications/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            viewed=False
+        )
+        self.payload.save()
+        self.payload.sources.add(self.second_creator, self.second_creator)
+        self.payload.save()
+
+        self.auth = {
+            "username": "test_dummy2",
+            "email": "test_dummy2@gmail.com",
+            "password1": "qwertyu12345",
```
Comment (chaitanya-cloudknox) [inline zubhub_backend/zubhub/notifications/tests/test_views.py]: Do not add username and passwords to the codebase. Get the username and passwords through environment variables

File: zubhub_backend/zubhub/notifications/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            "phone": "+57778899066",
+            "dateOfBirth": "1998-02-20",
+            "location": "Nigeria",
+            "bio": "ddd",
+            "subscribe": True,
+        }
+
+    def test_get_user_notifications(self):
+        rgst = self.client.post(reverse("creators:signup_creator"), self.auth, format="json")
+        res = self.client.get(reverse('notifications:user-notifications'))
```
Comment (chaitanya-cloudknox) [inline zubhub_backend/zubhub/notifications/tests/test_views.py:39]: Are there any steps required before completing registration (like email verification)

File: zubhub_backend/zubhub/notifications/tests/test_views.py (hunk 1 above)
Comment (iamhks) [reply zubhub_backend/zubhub/notifications/tests/test_views.py]: @chaitanya-cloudknox I am not adding credentials in the main project codebase but only in the test case files. Referring to the current test cases here: https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_backend/zubhub/creators/tests/test_views.py

File: zubhub_backend/zubhub/notifications/tests/test_views.py (hunk 2 above)
Comment (iamhks) [reply zubhub_backend/zubhub/notifications/tests/test_views.py:39]: @chaitanya-cloudknox I just checked and once you sign-up a new user it directly logs you in without any necessity of email-verification

**Comment by iamhks** [comment]
@tuxology Finally able to resolve the issue. I have updated the PR and here is the screenshot:

image

Please review and for the PUT api the reason to use param like that is because it was failing with status code 415 (Referred here: https://stackoverflow.com/questions/15153048/django-rest-framework-http-put-failing-with-415-on-django-1-5 , the created Notifications Object was not working with json.dumps(data)) Hence, made a request through local and added the same json body.

**Pull request merged**